### ISSUE TITLE
Digital ocean mysql doesn't support Chinese

### ISSUE THREAD

**Issue by litstrong** [litstrong/kkadda#73]

**Comment by litstrong** [comment]
http://www.cnblogs.com/livingintruth/p/3433259.html

**Comment by litstrong** [comment]
```
update api_userprofile set name='测试kkkkkk' where id=5;
```

**Comment by litstrong** [comment]
`show variables like '%char%;`

**Comment by litstrong** [comment]
```
show variables like '%char%';
```

**Comment by litstrong** [comment]
```
+--------------------------+----------------------------+
| Variable_name            | Value                      |
+--------------------------+----------------------------+
| character_set_client     | utf8                       |
| character_set_connection | utf8                       |
| character_set_database   | latin1                     |
| character_set_filesystem | binary                     |
| character_set_results    | utf8                       |
| character_set_server     | latin1                     |
| character_set_system     | utf8                       |
| character_sets_dir       | /usr/share/mysql/charsets/ |
+--------------------------+----------------------------+
```

**Comment by litstrong** [comment]
```
ALTER DATABASE databasename CHARACTER SET utf8 COLLATE utf8_general_ci;`
```

**Comment by litstrong** [comment]
`SHOW CREATE TABLE xxx`

**Comment by litstrong** [comment]
```
+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table     | Create Table                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| api_event | CREATE TABLE `api_event` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(256) NOT NULL,
  `description` varchar(256) NOT NULL,
  `date_created` datetime NOT NULL,
  `created_by_id` int(11) NOT NULL,
  `picture` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  KEY `api_event_created_by_id_bcbd50f7_fk_auth_user_id` (`created_by_id`),
  CONSTRAINT `api_event_created_by_id_bcbd50f7_fk_auth_user_id` FOREIGN KEY (`created_by_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=11 DEFAULT CHARSET=latin1 |
+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

**Comment by litstrong** [comment]
```
UPDATE api_userprofile SET name='中' WHERE id=6;
```

**Comment by litstrong** [comment]
http://stackoverflow.com/questions/3513773/change-mysql-default-character-set-to-utf-8-in-my-cnf

**Comment by litstrong** [comment]
This is fixed after uninstall mysql-5.5 and re-install mysql-5.5 and change all charset to utf-8, and then making migration.

The charset of table will be same with the charset defined in table.
